talent/application-process.md

```diff
@@ -0,0 +1,69 @@
+# Applying at source{d} 
```

Comment:
I find this document closely related to [guide/talent/hiring_process.md](https://github.com/src-d/guide/blob/master/talent/hiring_process.md) merged by https://github.com/src-d/guide/pull/92.

I understand this one is more "candidate-oriented", and the other one is more "company-oriented", but I think it could be weird if the same thing is explained in different ways or one of them could be even outdated.

Reply:
Yes, I agree. We need to check both guides to be consistent whenever we make any changes in the process.